### ISSUE TITLE
Don't display a server as outdated for optional updates

### ISSUE THREAD

**Issue by aStonedPenguin** [Facepunch/garrysmod-requests#1844]
### Details
The whole point of making an update optional is to not force all servers to restart when an update isn't relevant to them. By displaying the server as outdated this makes optional updates almost pointless as it makes players less likely to join your server and forces you to update.

**Comment by GitSparTV** [comment]
Why don't you follow updates/news? You will notice less people are joining your server

**Comment by GitSparTV** [comment]
oh ok, I see, it's about optional updates

**Comment by thegrb93** [comment]
Not a fan of the optional updates in general.

**Comment by viral32111** [comment]
Maybe only show the outdated server a day (or however long is better) after the update has been released to give servers time to update, even if its optional?

Even if the update is optional, I always kept my server on the latest version because it's just good practice.

**Comment by Corvezeo** [comment]
I wholeheartedly agree with this.

Please remove this from the game, we didn't need it for the last 7 years we don't need it now.

**Comment by IFNDirt** [comment]
Agreed. Showing out of date for an optional update is confusing and unnecessary. 

If you are wanting to show the game version so bad, just make it always display the server's current versionand only display out of date if it is a mandatory update.

**Comment by GarnetDarkRP** [comment]
I too don't quite understand why have optional updates, and then confuse the end user by mis-informing them that their client might not be running the same version as a server. It is ultimately up to us server hosts to decide whether or not it's necessary to push through an OPTIONAL update - one that was not deemed necessary/important enough to force by the developers.

I think all server owners and content providers alike would immensely appreciate if you focused your attention on patching up physics and engine-related incompatibilities that have plagued us for a decade now, as opposed to unnecessary features such as this.

**Comment by robotboy655** [comment]
All updates are mandatory, not all require network compatibility to be broken.
